### PR TITLE
[to-multiarch] Correct COMPUTE_NODE_TYPE on azure and drop jobs on arm64 nightly

### PR DESCRIPTION
**Pull request by shellyyang1989**
Closes [OCPQE-14462](https://issues.redhat.com/browse/OCPQE-14462)

The change includes:
1) Fix COMPUTE_NODE_TYPE on Azure profile
2) Drop nightly jobs on arm64 to multi-arch due to prow issue [ARMOCP-526](https://issues.redhat.com/browse/ARMOCP-526)

### CODE REVIEW

**Comment by shellyyang1989** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-arm64-nightly-azure-ipi-baselinecaps-v412-to-multiarch-p2-f14

**Comment by shellyyang1989** [comment]
cc @aleskandro @liangxia for review. Thanks!

**Comment by aleskandro** [comment]
> Lower test frequency due to the low priority and prow issue [ARMOCP-526](https://issues.redhat.com/browse/ARMOCP-526)

Jobs starting from an arm64 single-arch cluster should not be affected by ARMOCP-526, are they?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 1]
```diff
@@ -1035,7 +1035,7 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-baselinecaps-v412
-- as: azure-ipi-baselinecaps-v412-to-multiarch-p2-f14
+- as: azure-ipi-baselinecaps-v412-to-multiarch-p2-f360
```
[the remark this answers is not in the document]
Comment (shellyyang1989) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:1038]: ah, I forgot it. Let me update.

**Comment by shellyyang1989** [comment]
> > Lower test frequency due to the low priority and prow issue [ARMOCP-526](https://issues.redhat.com/browse/ARMOCP-526)
> 
> Jobs starting from an arm64 single-arch payload should not be affected by [ARMOCP-526](https://issues.redhat.com//browse/ARMOCP-526), are they?

Jobs starting from single-arch arm64 release and ending up with multi-arch release will be affected.

**Comment by aleskandro** [comment]
> > > Lower test frequency due to the low priority and prow issue [ARMOCP-526](https://issues.redhat.com/browse/ARMOCP-526)
> > 
> > Jobs starting from an arm64 single-arch payload should not be affected by [ARMOCP-526](https://issues.redhat.com//browse/ARMOCP-526), are they?
> 
> Jobs starting from single-arch arm64 release and ending up with multi-arch release will be affected.

It is affected for nightly jobs only right? Not the dev preview ones

**Comment by shellyyang1989** [comment]
> > > > Lower test frequency due to the low priority and prow issue [ARMOCP-526](https://issues.redhat.com/browse/ARMOCP-526)
> > > 
> > > 
> > > Jobs starting from an arm64 single-arch payload should not be affected by [ARMOCP-526](https://issues.redhat.com//browse/ARMOCP-526), are they?
> > 
> > 
> > Jobs starting from single-arch arm64 release and ending up with multi-arch release will be affected.
> 
> It is affected for nightly jobs only right? Not the dev preview ones

Exactly.

**Comment by shellyyang1989** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-azure-ipi-baselinecaps-v412-to-multiarch-p2-f30

**Comment by shellyyang1989** [comment]
INFO[2023-03-22T03:19:35Z] 

`Step azure-ipi-baselinecaps-v412-to-multiarch-p2-f30-ipi-install-install succeeded after 54m33s. Install passed.`

@aleskandro @liangxia it's ready for merge. Thank you!

**Comment by aleskandro** [comment]
Hi @shellyyang1989 given the timeline for having the arm-build01 cluster ready to onboard the QE jobs is not defined yet and that the jobs cannot run the migration when starting from arm single-arch payload, I'd propose to remove them and only keeping the dev-preview ones that, instead, work. Testing the installation without the upgrade/migration is useless in this context. So, if there are equivalent jobs using the dev-preview stream, we'd just delete these ones. If some non-equivalent jobs are defined in the nightly, let's add them to the dev-preview test config variant.

**Comment by shellyyang1989** [comment]
@aleskandro It's fine with me. I've dropped nightly arm64 to multi-arch jobs. Please take a look. Thank you for the suggestion!

**Pull request merged**